index.js

```diff
@@ -373,11 +373,19 @@ module.exports = function (config, deps) {
         .set(common.SESSION_TOKEN_HEADER, user.getUserToken())
         .end(
         function (err, res) {
+          console.log("err", err, "res", res);
```

Comment:
If we are going to have a console log message do we want something more descriptive, like console.log("Response from createDatasetForUser: err=", err, ", res=", res);`?